shared/chat/conversation/list-area/normal/container.js

```diff
@@ -53,6 +45,30 @@ let markedInitiallyLoaded = false
 
 export default compose(
   connect(mapStateToProps, mapDispatchToProps, mergeProps),
+  withStateHandlers(
+    {
+      _conversationIDKey: null,
+      _lastOrdinalTime: Date.now(),
+    },
+    {
+      // We don't let you try and load more within a second. Used to use the ordinal but maybe we just never want a super quick load
+      loadMoreMessages: (state, props) => ordinal => {
```

Comment:
i am plumbing through ordinal here. originally i was going to use it and only do this limiting if the ordinal is the same but there's also the case of accidentally loading and loading and loading if the scrolling is messed up (especially on desktop) so instead i'm making this more broad and just saying if you request a new load and its less than a second we just disallow it